src/targets/services/stats.js

```diff
@@ -0,0 +1,103 @@
+import fetch from 'node-fetch'
+import CozyClient from 'cozy-client'
+import { TRANSACTION_DOCTYPE, ACCOUNT_DOCTYPE } from 'doctypes'
+import { BankTransaction } from 'cozy-doctypes'
+import { groupBy } from 'lodash'
+import logger from 'cozy-logger'
+import {
+  getPeriod,
+  fetchTransactionsForPeriod,
+  getMeanOnPeriod
+} from 'ducks/stats/services'
+import { getCategoryId } from 'ducks/categories/helpers'
+
+global.fetch = fetch
+
+const log = logger.namespace('stats')
+
+const schema = {
+  transactions: {
+    doctype: TRANSACTION_DOCTYPE,
+    attributes: {},
+    relationships: {
+      account: {
+        type: 'belongs-to-in-place',
+        doctype: ACCOUNT_DOCTYPE
+      }
+    }
+  },
+  stats: {
+    doctype: 'io.cozy.bank.accounts.stats',
+    attributes: {},
+    relationships: {
+      account: {
+        type: 'has-one',
+        doctype: ACCOUNT_DOCTYPE
+      }
+    }
+  }
+}
+
+const client = new CozyClient({
+  uri: process.env.COZY_URL.trim(),
+  schema,
+  token: process.env.COZY_CREDENTIALS.trim()
+})
+
+BankTransaction.registerClient(client)
```

Comment:
I think we should do it on Document, since we can run into errors later when we find out that it was only registered on BankTransaction.